pkg/commands/venv.go

```diff
@@ -186,7 +195,7 @@ func GetOkctlEnvVars(opts OkctlEnvironment) (map[string]string, error) {
 }
 
 func getPathWithOkctlBinaries(opts OkctlEnvironment) string {
-	okctlPath := fmt.Sprintf("%s:%s", opts.KubectlBinaryDir, opts.AwsIamAuthenticatorDir)
+	okctlPath := fmt.Sprintf("%s:%s:%s", opts.KubectlBinaryDir, opts.KubensBinaryDir, opts.AwsIamAuthenticatorDir)
```

Comment:
This is starting to look like it should rather be written as strings.Join([]string{dir1, dir2, dir3}, ":")